networkit/_NetworKit.pyx

```diff
@@ -3021,6 +3021,17 @@ cdef extern from "cpp/io/GraphReader.h":
 		_GraphReader() nogil except +
 		_Graph read(string path) nogil except +
 
+cdef extern from "cpp/io/GraphReader.h" namespace "NetworKit::GraphReader":
+	cdef enum MultipleEdgesHandling "NetworKit::GraphReader::MultipleEdgesHandling":
+		DISCARD_EDGES,
+		SUM_WEIGHTS_UP,
+		KEEP_MINIUM_WEIGHT
+
+class _MultipleEdgesHandling(object):
```

Comment:
Shouldn't this be the class without "_" (and above with "_")? And why does this inherit from `object`, is this really needed? Then the import into the `graphio` module would also be cleaner.

Reply:
Agreed, that style would fit the .pyx way better: https://github.com/kit-parco/networkit/pull/169/commits/94564f867bcd3d25dd8a14cc9b11f860495f7680